### ISSUE TITLE
General Model

### ISSUE THREAD

**Issue by kosack** [VODF/vodf-docs#19]
From discussion on 2024-02-18:

![vodf_high_level_data_model_sketch](https://github.com/VODF/vodf-docs/assets/11677812/4aa92a7e-ff19-48d3-906a-ccbe900f9b3d)

**Comment by kosack** [comment]
new version:

![vodf_high_level_data_model_sketch_v3](https://github.com/VODF/vodf-docs/assets/11677812/161c4e31-3dba-4fe6-9ce5-3a097e5500bd)

**Comment by bkhelifi** [comment]
@kosack : are these scheme not obsolete, now?

**Comment by kosack** [comment]
Yes, I think this is replaced by https://github.com/VODF/vodf_schema/issues/5#issuecomment-2737123937
